core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -1644,6 +1656,7 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     for (_ <- 0 until numOfConsumersToAdd) {
       val newConsumer = new KafkaConsumer[Array[Byte], Array[Byte]](this.consumerConfig)
       consumerGroup += newConsumer
+      consumers += newConsumer
```

Comment:
I think this is not needed since consumerGroup is actually `consumers` for all callers.